### DISCUSSION TITLE
How to build a "release build" of Roslyn?

### DISCUSSION THREAD

**Discussion by TGJek** [dotnet/roslyn#74819] [Q&A]
I tried building Roslyn and I success but I got a "4.10.0-dev (<developer build>)" "developer build" version information, so how to build a "release build"?😇

**Comment by steveberdy** [comment]
What options did you pass when building?

**Reply by TGJek** [reply]
I just run build.sh without any other options.

**Reply by steveberdy** [reply]
Just use `./build.sh -pack`. If you run `./build.sh -help` it will show you the available options.
